.gitpod.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full:latest
```

Comment:
Would it make sense to use the python3.9 workspace image? (Ref: https://hub.docker.com/r/gitpod/workspace-python-3.9)

Reply:
What about  https://hub.docker.com/r/gitpod/workspace-python ?

Reply:
I would prefer to take the fixed version [1], as it provides us with less uncertainty about which python version is used. Image [2] currently runs python3.8 and I could not find information on how one gets notified about when they change interpreter version...

[1] https://hub.docker.com/r/gitpod/workspace-python-3.9
[2] https://hub.docker.com/r/gitpod/workspace-python

Reply:
Done